projects/plugins/jetpack/class.jetpack-modules-list-table.php

```diff
@@ -319,12 +326,13 @@ public function column_cb( $item ) {
 	 * @return string HTML.
 	 */
 	public function column_icon() {
-		$badge_text = $free_text = '';
+		$badge_text = '';
+		$free_text  = '';
 		ob_start();
 		?>
 		<a href="#TB_inline?width=600&height=550&inlineId=more-info-module-settings-modal" class="thickbox">
 			<div class="module-image">
-				<p><span class="module-image-badge"><?php echo $badge_text; ?></span><span class="module-image-free" style="display: none"><?php echo $free_text; ?></span></p>
+				<p><span class="module-image-badge"><?php echo esc_html( $badge_text ); ?></span><span class="module-image-free" style="display: none"><?php echo esc_html( $free_text ); ?></span></p>
```

Comment:
I don't see a point to either of these variables. They're both empty and echo without modification. Do you see something I may be missing?